src/lib/Embed.jsx

```diff
@@ -45,7 +51,11 @@ class Embed extends PureComponent {
   }
 
   embed(config) {
```

Comment:
Try to rename this ?

Reply:
create extends Embed in pbi, can we keep the same? the report component type is still 'embed' for all modes.